pyproject.toml

```diff
@@ -7,19 +7,20 @@ authors = ["davidtnfsh <davidtnfsh@gmail.com>"]
 [tool.poetry.dependencies]
 python = "^3.7"
 requests = "^2.23.0"
-apache-airflow = "^1.10.12"
-google-cloud-bigquery = "^1.27.2"
-searchconsole = {git = "https://github.com/joshcarty/google-searchconsole"}
+apache-airflow = {extras = ["google_auth"], version = "^1.10.13"}
+searchconsole = "^0.0.4"
+google-cloud-bigquery = "<2.8.0"
 
 [tool.poetry.dev-dependencies]
 safety = "^1.9.0"
 bandit = "^1.6.2"
+pytest-cov = "^2.8.1"
+black = "^19.10b0"
+mypy = "^0.770"
+pytest = "^5.4.1"
+flake8 = "^3.8.3"
+diagrams = "^0.17.0"
```

Comment:
What is `diagrams` for?

Reply:
long long time ago @tai271828 had recommended this tool at [discord](https://discord.com/channels/752904426057892052/755969596162441266/763802433817411624)

I integrated it into Airflow but now has been deprecated 😅 

will remove this one